common/cosmwasm-smart-contracts/multisig-contract/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 [dependencies]
 cw-utils = { workspace = true }
 cw3 = { workspace = true }
-cw3-fixed-multisig = { workspace = true, features = ["library"] }
+#cw3-fixed-multisig = { workspace = true, features = ["library"] }
```

Comment:
oh, I was looking for that, but I guess since everything is still green we didn't need it after all?